db/index.js

```diff
@@ -0,0 +1,17 @@
+require('dotenv').config();
+const Pool = require('pg-pool');
+const pool = new Pool ({
+  host: process.env.HOST,
+  port: 5432,
+  database: 'soundcrate',
+  user: process.env.USER,
```

Comment:
We should change these environment variables to DB_HOST, DB_USER and DB_PASSWORD just because we may need other 'user' and 'password' env variables.